<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_dx">
     <item>
      <widget class="QLabel" name="dxLabel">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>dx:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dxLineEdit">
       <property name="placeholderText">
        <string>mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_dz">
     <item>
      <widget class="QLabel" name="dzLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>dz:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dzLineEdit">
       <property name="placeholderText">
        <string>m</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_current">
     <item>
      <widget class="QLabel" name="currentLabel">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Current:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currentLineEdit">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="calculateButton">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_output">
     <item>
      <widget class="QLabel" name="outputLabel">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
